internal/clients/readiness.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 const (
-	readinessCheckTimeout = 5 * time.Second
+	DefaultReadinessCheckTimeout = 5 * time.Second
```

Comment:
This should have a comment IMHO. Linter should enforce it I believe, no sure why it doesn't.

Reply:
Added comments.